### DISCUSSION TITLE
How can i do a web scanning using a Nuclei ?

### DISCUSSION THREAD

**Discussion by themeswordpress** [projectdiscovery/nuclei#4466] [Q&A] [answered]
Dears 

How can i do a web scanning using a Nuclei ?

i.e crawling then web scanning like xss and sql so i need to enter my url then detect all web vuls

**Comment by MetzinAround** [comment, marked as answer]
Hi @themeswordpress . 

Our getting started material is available on blog, video, and the docs site! 

https://www.youtube.com/watch?v=wtHcXA2l9c4

https://blog.projectdiscovery.io/getting-started-with-projectdiscovery-in-linux-and-windows/

https://docs.projectdiscovery.io/getstarted-overview

Nuclei can't detect EVERY web vuln, but it can detect ones that we have a template for. I'd recommend reading more about how nuclei works here: https://blog.projectdiscovery.io/ultimate-nuclei-guide/
